inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -13,7 +13,7 @@
 
 
 def application_add(args: argparse.Namespace) -> str:
-    if (args.gpgKeyUri and args.gpgKeyName is None) or (args.gpgKeyName and args.gpgKeyUri is None):
+    if bool(args.gpgKeyUri) != bool(args.gpgKeyName):
```

Comment:
Consider using `all` or `any` with a generator expression for a more Pythonic check of GPG key parameters. [important]